docs/ManualDeploymentGuideScalarDLOnAzure-1.md

```diff
@@ -0,0 +1,217 @@
+# Deploy Scalar DL on Azure
+
+Scalar DL is a database-agnostic distributed ledger middleware containerized with Docker.
+It can be deployed on various platforms and is recommended to be deployed on managed services for production to achieve high availability and scalability, and maintainability.
+This guide shows you how to manually deploy Scalar DL on a managed database service and a managed Kubernetes service in Azure as a starting point for deploying Scalar DL for production.
+
+## What we create
+
+![image](images/azure-diagram.png)
+
+In this guide, we will create the following components.
+
+* An Azure Virtual Network associated with a Resource Group.
+* An AKS cluster with a Kubernetes node pool.
+* A managed database service.
+    * A Cosmos DB Account.
+* A Bastion instance with a public IP.
+* Azure container insights.
+
+## Prerequisites
+
+You must have required permissions as specified in [Cloud permissions document](./CloudPermissionsForScalarDLOnAKS.md) to create cloud resources to deploy Scalar DL on Azure.
+
+## Step 1. Configure your network
+
+Configure a secure network with your organizational or application standards.
+Scalar DL handles highly sensitive data of your application, so you should create a highly secure network for production.
+This section shows how to configure a secure network for Scalar DL deployments.
+
+### Requirements
+
+* You must create a virtual network with a subnet for bastion.
+* You must create 2 subnets with the prefix of at least `/22` for AKS, one subnet must be created with the name `k8s_ingress` to create an envoy load balancer.
+
+### Recommendations
+
+* You should create a bastion server to manage the Kubernetes cluster for production.
+
+### Steps
+
+* Create an Azure virtual network on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/virtual-network/quick-create-portal) with the above requirements and recommendations.
```

Comment:
Need to create a Bastion, right?
It's inconsistent.

Reply:
OK. Added one more step to create bastion in 4791dc7